core/src/entities/cyclonedx/models/bom.rs

```diff
@@ -0,0 +1,138 @@
+use serde::{Deserialize, Serialize};
+
+use crate::entities::cyclonedx::Component;
+use crate::entities::sboms::CdxFormat;
+use crate::Error;
+
+/*
+ * Generated by: https://openapi-generator.tech
+ */
+
+pub fn default_id() -> String {
+    String::from("")
+}
+
+#[derive(Clone, Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct Bom {
```

Comment:
Well, this is CycloneDx's model from their JSON spec. Not ours. I wouldn't want to change that to be honest. I "think" CycloneDx can actually be generated at build time if we move some of my helper functions to a different place, and we could end up ignoring this folder from a source control perspective. At least, it would be nice if we could do something like that.

Reply:
Got it. :+1: